.changeset/heavy-crabs-share.md

```diff
@@ -0,0 +1,5 @@
+---
+'@getodk/web-forms': patch
+---
+
+Fix error style for required questions
```

Comment:
I like the PR title better! "Show error highlight on required questions after clearing answer"